docs/src/modify.md

```diff
@@ -31,4 +31,8 @@ using MarketData
 rename(cl, :Close′)
 rename(cl, [:Close′])
 rename(ohlc, [:Open′, :High′, :Low′, :Close′])
+rename(ohlc, :Open => :Open′)
+rename(ohlc, Dict(:Open => :Open′, :Close => :Close′))
```

Comment:
An other approach could be to support multiple pairs for replacement

    rename(ohlc, :Open => :Open′, :Close => :Close′)

or

    rename(ohlc, Dict(:Open => :Open′, :Close => :Close′)...)

it seems to be where `replace` function should be going according

`https://github.com/JuliaLang/julia/issues/29849#issuecomment-434321585`

Reply:
+ 1 for `Vararg` pairs, so we do not need to support Dict explicitly,
just show the usage in doc is enough.

Reply:
Absolutely so we could do

    rename(ohlc, Dict(:Open => :Open′, :Close => :Close′)...)

which is enough